patches_for_WebRTC_org/m84/src/modules/audio_device/win/audio_device_core_win.cc

```diff
@@ -635,6 +635,8 @@ struct AudioDeviceHelper : public DeviceHelper<DEVICE_CLASS> {
       CloseHandle(_hThread);
       _hThread = nullptr;
       _transporting = false;
+      _transportInitialized = false;
```

Comment:
Can these be set to false on line 630 to avoid setting the same value in two places? Or would that introduce a potential race condition?

Reply:
I feel that there's the risk of a race condition, as if you see `_transporting` was set to false in both places even before my change.